reactos/drivers/usb/usbport/usbport.c

```diff
@@ -2644,8 +2669,7 @@ USBPORT_Dispatch(IN PDEVICE_OBJECT DeviceObject,
                        IoStack->MinorFunction);
             }
 
-            Status = STATUS_INVALID_DEVICE_REQUEST;
-            Irp->IoStatus.Status = Status;
+            Irp->IoStatus.Status = STATUS_INVALID_DEVICE_REQUEST;
```

Comment:
This change is odd. Now you'll return STATUS_SUCCESS, which is not the status you completed the IRP with.